spec/cucumber/core_spec.rb

```diff
@@ -216,9 +271,10 @@ def expect_tag_excess(error_message)
               end
             end
           end
-          report = Core::Report::Summary.new
 
-          execute [gherkin], report, [Core::Test::Filters::ActivateStepsForSelfTest.new]
+          event_bus = Core::EventBus.new
```

Comment:
Earlier in this file (in `execute [gherkin], [Core::Test::Filters::ActivateStepsForSelfTest.new] do |events|`) the `EventBus` was named `events`, but here it is called `event_bus`, I find it confusing.